src/net/tls.cc

```diff
@@ -962,20 +975,38 @@ void tls::reloadable_credentials<tls::certificate_credentials>::rebuild(const cr
     this->_impl = std::move(tmp->_impl);
 }
 
+template <>
+const tls::certificate_credentials &tls::reloadable_credentials<tls::certificate_credentials>::as_certificate_credentials() const noexcept {
+    return *this;
```

Comment:
Our coding style attaches the reference sigil to the type, not the function name.